### DISCUSSION TITLE
AC led panel control

### DISCUSSION THREAD

**Discussion by alloploid** [nVuln/homebridge-lg-thinq#126] [Q&A]
Is this feature working for you? 
I have LG Artcool Mirror  (Platform 1) and the light switch doesn't make any difference.

`"platforms": [
        {
            "platform": "LGThinQ",
            "country": "PL",
            "language": "en-US",
            "refresh_interval": 60,
            "auth_mode": "account",
            "username": "*****",
            "password": "******",
            "thinq1": true,
            "devices": [
                {
                    "id": "d27b3ec0-7149-11d3-80ac-044eaf393993",
                    "name": "Klima Sypialnia",
                    "type": "AC",
                    "washer_door_lock": false,
                    "ac_mode": "BOTH",
                    "ac_swing_mode": "BOTH",
                    "ac_temperature_sensor": true,
                    "ac_led_control": true
                }
            ]
        }
    ]`

**Comment by nVuln** [comment]
> Is this feature working for you? I have LG Artcool Mirror (Platform 1) and the light switch doesn't make any difference.
> 
> `"platforms": [ { "platform": "LGThinQ", "country": "PL", "language": "en-US", "refresh_interval": 60, "auth_mode": "account", "username": "*****", "password": "******", "thinq1": true, "devices": [ { "id": "d27b3ec0-7149-11d3-80ac-044eaf393993", "name": "Klima Sypialnia", "type": "AC", "washer_door_lock": false, "ac_mode": "BOTH", "ac_swing_mode": "BOTH", "ac_temperature_sensor": true, "ac_led_control": true } ] } ]`

some model do not support control led panel, can you control it via native LG Thinq app ?

**Reply by alloploid** [reply]
Native LG Thinq app - no,
but it works via remote controller or Sensibo.

**Reply by nVuln** [reply]
if native app can't do it, this plugin can't too
remote or sensibo is just locally method
